hooks/focus.js

```diff
@@ -1,15 +1,16 @@
-import {useRef, useEffect} from 'react'
+import {useState, useRef, useEffect} from 'react'
 
-function useFocus(isFocus) {
-  const focusedElement = useRef()
+function useFocus({autofocus = false}) {
```

Comment:
Il n'y qu'une seule prop, il n'est peut-être nécessaire d'envoyer un objet.

```suggestion
function useFocus(autofocus = false) {
```